deploy/cephfs/kubernetes/v1.13/csi-nodeplugin-rbac.yaml

```diff
@@ -46,8 +48,82 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: cephfs-csi-nodeplugin
+    # replace with non-default namespace name
     namespace: default
```

Comment:
not release to this PR, can you make namespace as  configurable in  next PR?

Reply:
Since these are static manifests, it's unfortunately very tricky to achieve this without templating like helm charts or scripting with `envsubst` or `sed`.

Reply:
my bad, I thought these changes are in helm charts :D . I will add a  script or steps to do it. Thanks

Reply:
Should I remove my changes related to namespaces from files under `deploy/` or leave them for now?

Reply:
let's keep them :+1: